net/server.go

```diff
@@ -447,3 +493,153 @@ func (s *server) SendPubSubMessage(
 	}
 	return t.Publish(ctx, data)
 }
+
+// hasAccess checks if the requesting peer has access to the given cid.
+func (s *server) hasAccess(p peer.ID, c cid.Cid) bool {
```

Comment:
suggestion: took me awhile to figure out that this is used by bitswap. Maybe add a comment?

Reply:
will do.

Reply:
✅